pkg/patterns/addon/pkg/utils/helpers.go

```diff
@@ -9,48 +9,71 @@ import (
 	addonsv1alpha1 "sigs.k8s.io/kubebuilder-declarative-pattern/pkg/patterns/addon/pkg/apis/v1alpha1"
 )
 
-func GetCommonVersion(instance runtime.Object) (string, error) {
+func genError(v runtime.Object) error {
+	return fmt.Errorf("instance %T is not addonsv1alpha1.CommonObject or unstructured", v)
```

Comment:
I probably would have just inlined this function each time, but I guess it's nice that it is consistent! :+1:

Reply:
Thanks. I could change it if that would be more concise.